STORM-3224: Fix FLUX YAML Viewer icon location/position on UI page

kishorvpatil opened the pull request.

revans2 commented:
Sorry to pull my +1 back.  On both Firefox and Chrome it looks off to me.  Could you take a look at this again?

![ui](https://user-images.githubusercontent.com/3441321/45642002-a151be00-ba7c-11e8-9101-161114487d54.png)

kishorvpatil commented:
OMG, my bad.. looks like the span tag is added twice. Let me fix it..

kishorvpatil commented:
<img width="1430" alt="fixed-ui-page" src="https://user-images.githubusercontent.com/6090397/45646222-afa9d500-ba90-11e8-9651-56b153e7ce42.png">

@revans2 I fixed the position and removed duplicate space entries showing FLUX image icon.
